pkg/antctl/command_definition.go

```diff
@@ -639,18 +646,26 @@ func (cd *commandDefinition) newCommandRunE(c AntctlClient, out io.Writer) func(
 
 // applyFlagsToCommand sets up args and flags for the command.
 func (cd *commandDefinition) applyFlagsToCommand(cmd *cobra.Command) {
-	var hasFlag bool
+	var hasArg bool
 	for _, flag := range cd.getEndpoint().flags() {
 		if flag.arg {
 			cmd.Args = cobra.MaximumNArgs(1)
 			cmd.Use += fmt.Sprintf(" [%s]", flag.name)
 			cmd.Long += fmt.Sprintf("\n\nArgs:\n  %s\t%s", flag.name, flag.usage)
-			hasFlag = true
+			hasArg = true
 		} else {
-			cmd.Flags().StringP(flag.name, flag.shorthand, flag.defaultValue, flag.usage)
+			if flag.isBool {
+				defaultValue := false
+				if flag.defaultValue == "true" {
+					defaultValue = true
+				}
```

Comment:
I think there is potential for improvement for `flagInfo` in the future. But for this PR, I would recommend just ignoring the `defaultValue` field for boolean flags, and adding a command to the `isBool` field in the `flagInfo` struct to indicate that the flag always defaults to false when omitted

Reply:
Make sense, ignore the defaultValue and add a comment.